### PR TITLE
Completed

### PR DESCRIPTION
**Pull request by theaubmov**
## Summary by CodeRabbit

- **New Features**
	- Enhanced logging and modification capabilities for data store event handling.
	- Improved data store selection logic by incorporating unique identifiers and classes.

- **Bug Fixes**
	- Adjusted data store definitions and tests to align with new ID and type specifications.

- **Tests**
	- Updated tests to accommodate changes in data store value handling and identity management.

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -187,10 +187,11 @@ bpmnModeler.on('spiff.dmn_files.requested', (event) => {
 });
 
 bpmnModeler.on('spiff.data_stores.requested', (event) => {
+  console.log(event)
```
Comment (burnettk) [inline app/app.js:190]: shall we kill the console?

File: app/spiffworkflow/DataStoreReference/propertiesPanel/DataStoreSelect.js [hunk 2]
```diff
@@ -21,13 +21,21 @@ export function DataStoreSelect(props) {
...
+    const dtRef = element.businessObject.dataStoreRef;
+    return dtRef
+      ? `${dtRef.id}___${dtRef.name}`
       : '';
   };
 
   const setValue = (value) => {
-    if (!value || value == '') {
+
+    console.log('elements', element)
```
Comment (burnettk) [inline app/spiffworkflow/DataStoreReference/propertiesPanel/DataStoreSelect.js:32]: another console

**Review by burnettk — APPROVED** [review]

File: app/app.js (hunk 1 above)
Comment (theaubmov) [reply app/app.js]: yup,

**Comment by theaubmov** [comment]
@burnettk I deleted the console logs

**Pull request merged**